.github/workflows/pytest.yml

```diff
@@ -0,0 +1,16 @@
+name: PyTest
```

Comment:
This code already merged as part of:
https://github.com/redhat-beyond/commune/pull/34

please rebase